.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  parser: 'babel-eslint',
+  env: {
+    browser: true,
+    es6: true
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'prettier',
+    'plugin:prettier/recommended'
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly'
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module'
+  },
+  settings: {
+    react: {
+      version: '16.8.3'
+    }
+  },
+  plugins: ['prettier', 'react', 'react-hooks'],
+  rules: {
+    'prettier/prettier': 'error',
+    'react-hooks/rules-of-hooks': 'error',
+    'react-hooks/exhaustive-deps': 'warn',
```

Comment:
This should also be `error` - but if we're going to fix the issues in a separate PR we can set it to `error` then.

Reply:
@bmullan91 - I haven't set this to `error` in this PR because it wanted me to change the deps in those two files. Changing as advised by the linter broke the tests or removed the deps array completely. Might be best for someone familiar with that behaviour to take a look at those warnings.